libs/pando/ui-sheet/src/components/DocumentDisplay.tsx

```diff
@@ -136,3 +144,56 @@ export function HeaderDisplay({
     </>
   )
 }
+
+function ConditionalDisplay({
+  conditional,
+  bgt = 'normal',
+}: {
+  conditional: Conditional
+  bgt?: CardBackgroundColor
+}) {
+  const { header, fields } = conditional
+  return (
+    <CardThemed bgt={bgt}>
+      {!!header && <HeaderDisplay header={header} />}
+      <ConditionalSelector conditional={conditional} />
+      {!!fields && <FieldsDisplay bgt={bgt} fields={fields} />}
+    </CardThemed>
+  )
+}
+function ConditionalSelector({ conditional }: { conditional: Conditional }) {
+  return <BoolConditional conditional={conditional} />
+}
+function BoolConditional({ conditional }: { conditional: Conditional }) {
+  const calc = useContext(CalcContext)
+  const { label, badge } = conditional
+
+  const conditionalValue = calc?.compute(
+    read(
+      {
+        et: 'own',
+        qt: 'cond',
+        sheet: conditional.metadata.sheet,
+        q: conditional.metadata.name,
+        src: 'all',
+        dst: calc.cache.tag.src,
+      },
+      'max'
+    )
+  ).val
+  return (
+    <Button
+      fullWidth
+      size="small"
+      sx={{ borderRadius: 0 }}
+      color={conditionalValue ? 'success' : 'primary'}
+      // onClick={() => setConditional(conditionalValue ? undefined : stateKey)}
```

Comment:
Will likely have to propdrill a generalized function all the way from `DocumentDisplay`